pta-intelligent-search-metadata-extractor/src/main/java/fi/maanmittauslaitos/pta/search/metadata/json/extractor/GeographicBoundingBoxCKANCustomExtractor.java

```diff
@@ -48,7 +42,11 @@ public Object process(DocumentQuery documentQuery, QueryResult queryResult) thro
 			ret[3] = Ys.stream().max(Double::compareTo).orElseThrow(MissingCoordException::new);
 
 		} catch (RuntimeException e) {
```

Comment:
Here be dragons

Reply:
https://github.com/nlsfi/pta-intelligent-search/pull/15#discussion_r387926520